autoload/ollama/setup.vim

```diff
@@ -412,21 +421,18 @@ use_venv = vim.eval('g:ollama_use_venv') or 0
 
 # Should we use a venv?
 if use_venv:
-    # Create default venv path
     venv_path = os.path.join(os.environ['HOME'], '.vim', 'venv', 'ollama')
-    # Check if the venv path exists
     if os.path.exists(venv_path):
-        #print('Found venv:', venv_path)
-
         venv_bin = os.path.join(venv_path, 'bin', 'python3')
-        venv_site_packages = os.path.join(venv_path, 'lib', f'python{sys.version_info.major}.{sys.version_info.minor}', 'site-packages')
-
-        # Ensure the virtual environment's site-packages is in sys.path
+        venv_site_packages = os.path.join(
+            venv_path,
+            'lib', f'python{sys.version_info.major}.{sys.version_info.minor}',
+            'site-packages'
+        )
         if venv_site_packages not in sys.path:
-            #print(f'Adding venv site-packages to path: {venv_site_packages}')
             sys.path.insert(0, venv_site_packages)
     else:
-        print('Venv not found: '. venv_path)
+        print('Venv not found: ' + venv_path)
```

Comment:
*minor*

This has been fixed already.
It seems that this project uses merges for PRs, so maybe it does not matter much.
But if you rebase this minor fix should disappear.